config/webpack.common.js

```diff
@@ -147,8 +152,14 @@ module.exports = (env) => {
               // In real prod mode, populate window._env at run time with express
               filename: 'index.html.ejs',
               template: `!!raw-loader!${path.resolve(__dirname, '../src/index.html.ejs')}`,
-            }
-      ),
+            }),
+        favicon: path.resolve(
+          __dirname,
+          process.env.BRAND_TYPE === 'RedHat'
+            ? '../src/favicon-redhat.ico'
+            : '../src/favicon-konveyor.ico'
```

Comment:
@mturley 
I noticed in the sprint 195 demo that you are using the Konveyor icon rather than the Forklift icon. Will the Konveyor icon be used for all the UIs?

Reply:
@apinnick I was only replacing Red Hat branding with Konveyor branding in the upstream, which I figured was equivalent because in the upstream builds we're using Red Hat icons (and not Migration Toolkit icons). It's a good question though, maybe we should be using product-specific favicons.

Either way though the product logo could probably go on the welcome page instead of the generic migration icon.
@vconzola do you have thoughts on these, or know who we should ask?